alephium/contracts/token_bridge/token_bridge.ral

```diff
@@ -129,26 +129,15 @@ Contract TokenBridge(
         return current
     }
 
-    @using(preapprovedAssets = true)
-    pub fn createWrappedAlphPool(payer: Address, createContractAlphAmount: U256) -> () {
-        let encodedFields = encodeToByteVec!(
-            selfContractId!(),
-            localChainId,
-            wrappedAlphId,
-            0,
-            0
-        )
-        let path = Path.TokenPool ++ u256To2Byte!(localChainId) ++ wrappedAlphId
-        copyCreateSubContract!{payer -> createContractAlphAmount}(
-            path,
-            tokenBridgeFactory.getWrappedAlphPoolTemplateId(),
-            encodedFields
-        )
-    }
-
     @using(preapprovedAssets = true)
     pub fn createLocalTokenPool(localTokenId: ByteVec, payer: Address, createContractAlphAmount: U256) -> () {
-        assert!(tokenRemaining!(payer, localTokenId) == 1, ErrorCodes.TokenNotExist) // make sure that the token exists
+        let isWrappedAlph = localTokenId == wrappedAlphId
+        let templateContractId =
+            if (isWrappedAlph) tokenBridgeFactory.getWrappedAlphPoolTemplateId()
+            else tokenBridgeFactory.getLocalTokenPoolTemplateId()
```

Comment:
[Nit]: move this line just above the following `copyCreateSubContract`, so the variable is created when it's used and review will be slightly easier.